_install-and-configure/install-opensearch/tar.md

```diff
@@ -94,6 +94,12 @@ An OpenSearch node configured by the demo security script is not suitable for a
    ```
    {% include copy.html %}
 
+1. Set an initial admin password variable through the `OPENSEARCH_INITIAL_ADMIN_PASSWORD` environment variable.
+   ```bash
+   export OPENSEARCH_INITIAL_ADMIN_PASSWORD=< Admin password >
+   ```
+   {% include copy.html %}
+
 1. Run the OpenSearch startup script with the security demo configuration.
```

Comment:
this step should be numbered as 2

Reply:
All of them are 1

Reply:
This is correct, I believe the numbers are handled on the website side.